website/src/categories/components/avatar/code.md

```diff
@@ -6,30 +6,45 @@ eleventyNavigation:
   parent: Avatar
   key: AvatarCode
 ---
-<section>
+<section>  
+<div class="ds-example" style="justify-content: space-evenly">
+  <sl-avatar display-name="Anna Jenssen" picture-url="/assets/images/components/avatar/toa-heftiba-ANNsvl-6AG0-unsplash.jpg" size="3xl" orientation="vertical"></sl-avatar>
+  <sl-avatar display-name="Tim Jenssen" picture-url="/assets/images/components/avatar/xia-yang-AGGA9LH3FLo-unsplash.jpg" size="xl" status="success">Class: 1B</sl-avatar>
+
+  <small style="position: absolute; bottom:0; right:var(--scale-150-scale)">
+
+  Photos by [Xia Yang](https://unsplash.com/@imrxia?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash)
+  and [Toa Heftiba](https://unsplash.com/@heftiba?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash)
+
+  </small>
+</div>
 
-## Avatar code
+<div class="ds-code">
 
-Avatar examples
+  ```html
```

Comment:
Maybe we can show different examples in the `code` tab than in the `usage` tab like we are showing different ones in other places (other components documentation)?
E.g. we can show a version with initials only?